.github/workflows/main.yml

```diff
@@ -142,16 +142,32 @@ jobs:
         sudo apt install libxss1 libgstreamer-plugins-base1.0-0 libgstreamer1.0-0
         sudo apt install patchelf binutils cmake pkg-config libssl-dev libxss-dev libxmu-dev
         sudo apt install dbus libfreetype6 fontconfig libx11-6 libx11-xcb1 libgl1-mesa-dev libnss3 libasound2 libxcomposite1 libxcursor1 libxi6 libxtst6 wget
-    - name: Fetch and install binary Qt
+    - name: Upgrade GCC
       run: |
-        bash ./dist/linux/install-qt.sh 5.12.5
+        sudo apt-get install -y software-properties-common
+        sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+        sudo apt update
+        sudo apt install g++-7 -y
+        sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 60 --slave /usr/bin/g++ g++ /usr/bin/g++-7
+        sudo update-alternatives --config gcc
+    - name: Cache Qt
+      id: cache-qt
+      uses: actions/cache@v1
+      with:
+        path: ../Qt
+        key: QtCache
```

Comment:
The key doesn't depend on the Qt version, so when the new version of Qt is released, the cache of the old version would still be used. Could this be an issue?

Reply:
We would have to manually bump the Qt version to actually upgrade it. The action installs 5.12 series Qt by default and the company stopped releasing LTS z-versions. If we manually change the Qt version, we should probably change the cache name as well. But yeah, good point, I'll change the name of the cache so it's clear we should change it when changing Qt version.